contracts/mocks/docs/access-control/AccessControlERC20MintBase.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {AccessControl} from "../../../access/AccessControl.sol";
+import {ERC20} from "../../../token/ERC20/ERC20.sol";
+
+contract AccessControlERC0Mint is ERC20, AccessControl {
+    // Create a new role identifier for the minter role
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+
+    error CallerNotMinter(address caller);
+
+    constructor(address minter) ERC20("MyToken", "TKN") {
+        // Grant the minter role to a specified account
+        _grantRole(MINTER_ROLE, minter);
+    }
```

Comment:
This is not good practice:
- there is not `DEFAULT_ADMIN_ROLE` granted
- the minter can renonce its role, but not transfer it.

I'd arguen line 15 should grant the DEFAULT_ADMIN_ROLE (either instead or in addition to the MINTER_ROLE)